core/pom.xml

```diff
@@ -353,7 +353,7 @@
             org.wso2.carbon.messaging.*,
         </import.package>
         <carbon.component>
-            startup.listener;componentName="wso2-microservices-server";requiredService="org.wso2.msf4j.Microservice,org.wso2.msf4j.Interceptor",
```

Comment:
As of my understanding MSF4J core does not need to wait for any type of interceptors (since there can be services written without using any interceptors. Deprecated interceptors are used only in MSF4J analytics components and the relevant samples. In there this line has been not changed.